src/Crypto/RNCryptor/V3/Decrypt.hs

```diff
@@ -94,6 +92,13 @@ removePaddingSymbols input =
   in B.take (B.length input - fromEnum lastWord) input
 
 --------------------------------------------------------------------------------
+decrypt_ :: AES128 -> ByteString -> ByteString -> ByteString
```

Comment:
Good catch!   Fixed.   Crypto issues like this confuse me though.  Why did it work then, when I used AES128 and checked I could decrypt successfully with the Swift implementation, I wonder?